Cargo.toml

```diff
@@ -20,6 +20,7 @@ once_cell = "1.12.0"
 thiserror = "1.0.30"
 camino = { version = "1.0.7", features = ["serde1"] } 
 ahash = { version = "0.7.6", features = ["std"] }
+minreq = { version = "=2.2.1", features = ["https", "json-using-serde"] }
```

Comment:
"optional = true" + add to the updater feature flag (bottom of the file) and rename it to "online"